### ISSUE TITLE
IDLE: understand why outlook do IDLE without selected mailbox

### ISSUE THREAD

**Issue by chibenwa** [linagora/james-project#5320]
Seen on CNB prod

![image](https://github.com/user-attachments/assets/c9a4edcb-b7b0-42d5-8281-e44ab9a409dd)

several IDLE are issued, first without selected mailbox then with selected mailbox.

Specific to outlook.

Goal would be to review the full list of commands issued by outlook to investigate if this behaviour is a bug on our side...

**Comment by vttranlina** [comment]
Does the information in https://github.com/linagora/james-project/issues/5321 address this ticket?

IDLE is allowed in both Authenticated and Selected state

**Comment by chibenwa** [comment]
> Does the information in https://github.com/linagora/james-project/issues/5321 address this ticket?

Not really. Understanding why something happens is not equivalent to reject it.

> IDLE is allowed in both Authenticated and Selected state

And the spec do not specify what to do in non selected state... It's ambiguous, we shall thus see what others do!

**Comment by vttranlina** [comment]
>  we shall thus see what others do!

Gmail and Cysrus support `IDLE` at Authenticated State. 
But just `idling` and do nothing

**Comment by chibenwa** [comment]
> But just idling and do nothing

So James behaviour is normal!

Please let's add a test for this behaviour to be explicitly tested!

Also, we should in depth review the outlook exchanges!
